### PR TITLE
Python: update highlights

### PR DESCRIPTION
**Pull request by stsewd**
- Fix "as" outside imports
- Add builtin classes and constants

### CODE REVIEW

File: queries/python/highlights.scm [hunk 1]
```diff
@@ -13,10 +13,22 @@
 ((identifier) @constant.builtin
  (#match? @constant.builtin "^__[a-zA-Z0-9_]*__$"))
 
+((identifier) @constant.builtin
+ (#vim-match?
+  @constant.builtin
+  ; https://docs.python.org/3/library/constants.html
+  "^(NotImplemented|Ellipsis|quit|exit|copyright|credits|license)$"))
```
Comment (theHamsta) [inline queries/python/highlights.scm:20]: Can you indent it like that? Lisp convention is usually to have function arguments align with their indentation and to use `;;` when comment refers to next line of the same sexpr.
```scheme
((identifier) @constant.builtin
 (#vim-match?  @constant.builtin
                        ;; https://docs.python.org/3/library/constants.html
                        "^(NotImplemented|Ellipsis|quit|exit|copyright|credits|license)$"))
```

**Comment by tjdevries** [comment]
I'd be interested in a speed comparison of this vs. `#contains?`

File: queries/python/highlights.scm (hunk 1 above)
Comment (stsewd) [reply queries/python/highlights.scm]: Done!

**Comment by stsewd** [comment]
> I'd be interested in a speed comparison of this vs. `#contains?`

@tjdevries I tested it before, but it doesn't match the exact string. For example `(#contains? @foo "foo")` would match `foo` and `foobar`. Guessing that's a bug?

**Comment by tjdevries** [comment]
@vigoux thoughts on above?

**Comment by vigoux** [comment]
I think that it's a feature as we search for substrings, we should extend the `eq?` predicate otherwise to accept multiple arguments

**Pull request merged**